### ISSUE TITLE
Fedora 33 Build on Fedora 33 fails.

### ISSUE THREAD

**Issue by mottsen** [OSInside/kiwi#1772]
## Problem description

Builds fail on Fedora 33 when building Fedora 33 image.  I did find a [bug report](https://github.com/openSUSE/libsolv/issues/412) for libsolv that seemed to possibly be related, but the fix for that should have gone out in libsolv 0.7.17, but appears to be the version that I'm running on this Fedora VM (0.7.17-1.fc33).

## Expected behaviour

Build is successful.

## Steps to reproduce the behaviour

1. Install fresh Fedora 33 workstation system.
2. Do a full dnf update
3. Install python3-kiwi
4. Clone the [kiwi-descriptions](https://github.com/OSInside/kiwi-descriptions) repo
5. Attempt to build the Fedora 33 description. (Note I did this as root to rule out permission issues)
6. See this error: `KiwiCommandError: rpm: stderr: error: can't create transaction lock on /root/fedora/build/image-root/var/lib/rpm/.rpm.lock (No such file or directory)`
7. Build same description using universal boxbuild targeting the same directory and watch it succeed.

## OS and Software information
+ KIWI version: 9.23.20-2.fc33 (I also installed 9.23.20-1.1 from [here](https://download.opensuse.org/repositories/Virtualization:/Appliances:/Builder/Fedora_33/) and got the same results)
+ Operating system: Fedora 33
+ OBS version: N/A

**Comment by schaefi** [comment]
Thanks for reporting. The universal box is build against Fedora_32 and so it seems related to the change you mentioned.
At the moment I have no idea what we could do in the kiwi code to fix this

**Comment by schaefi** [comment]
I deleted the prio label if you don't mind. We use the prio labels in the sprint meetings to agree for a prio and that this will be worked on. This issue was not discussed in any sprint meeting and it also seems not to be a low hanging fruit.

**Comment by schaefi** [comment]
This is now fixed. With the rebuild of the fedora box including fedora updates plus the fix in #1773 I could successfully build as follows:

```
kiwi-ng system boxbuild --box fedora --box-debug -- --description ~ms/Project/kiwi-descriptions/fedora/x86_64/fedora-33.0/ --target-dir /tmp/mytest
```

Please re-open if it does not work for you

Thanks
